lib/main.dart

```diff
@@ -106,20 +109,23 @@ class _SplashScreenState extends State<SplashScreen> with SingleTickerProviderSt
     );
   }
 
+// Carlo
```

Comment:
Disposing _fadeController is good for preventing memory leaks and having efficient resource management.